python/paddle/fluid/dygraph/varbase_patch_methods.py

```diff
@@ -50,14 +50,16 @@ def _to_static_var(self, to_parameter=False, **kwargs):
                     static_var = var_base._to_static_var()
 
         """
+        attr_not_need_keys = ['grad']
```

Comment:
Maybe we can consider add `grad` for Variable to make `Variable` and `Tensor` consistent.

Reply:
grad for Variable is fake_interface_only